download_pdfs.py

```diff
@@ -1,53 +1,50 @@
-import os
+import os, yaml
```

Comment:
PEP 8 [recommends](https://peps.python.org/pep-0008/#imports) placing individual imports on separate lines.  (I'm surprised that Flake8 let you get away with this.  😉)

Reply:
It complained this time, fixed